utils/cliutils/commandsflags.go

```diff
@@ -1244,6 +1245,10 @@ var flagsMap = map[string]cli.Flag{
 		Name:  xrOutput,
 		Usage: "[Default: table] Defines the output format of the command. Acceptable values are: table, json, simple-json and sarif.` `",
 	},
+	BypassArchiveLimits: cli.BoolFlag{
+		Name:  BypassArchiveLimits,
+		Usage: "[Default: false] Set to true to bypass JFrog Xray Indexer archive limits.` `",
```

Comment:
...to bypass JFrog Xray Indexer archive limits.... --> ...to bypass the indexer-app archive limits....